docs/config.md

```diff
@@ -100,6 +95,34 @@ it, you can build Riot, but run
 Alternatively, you can edit the `og:image` meta tag in `index.html` directly
 each time you download a new version of Riot.
 
+Identity servers
+================
+
+The identity server is used for inviting other users to a room via third party
+identifiers like emails and phone numbers. It is not used to store your password
+or account information.
+
+As of Riot 1.4.0, all identity server functions are optional and you are
+prompted to agree terms before data is sent to the identity server.
+
+Riot will check multiple sources when looking for an identity server to use in
+the following order of preference:
+
+1. The identity server set in the user's account data
+   * For a new user, no value is present in their account data. It is only set
```

Comment:
users can also explicitly pick "none" by disconnecting. Not sure if it's worth mentioning here.

Reply:
Right, my thoughts while writing this were that mentioning the disconnecting would be too much info to process, so I left it out for now.